test/Microsoft.NET.Build.Tests/GivenThatWeWantToVerifyNuGetReferenceCompat.cs

```diff
@@ -11,22 +11,12 @@ public GivenThatWeWantToVerifyNuGetReferenceCompat(ITestOutputHelper log) : base
         {
         }
 
+        // Reduced set of test cases that still verify the compatibility matrix
```

Comment:
nit:
This comment seems relevant for the PR but would be confusing to me if I saw it in the codebase with no context in 10 years.